src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -26,4 +26,5 @@ export default Object.freeze({
   gibctEstimateYourBenefits: 'gibctEstimateYourBenefits',
   form526OriginalClaims: 'form526OriginalClaims',
   vaViewDependentsAccess: 'vaViewDependentsAccess',
+  allowOnline1010cgSubmissions: 'allowOnline1010cgSubmissions',
```

Comment:
The API already had this flag registered for this use case, https://github.com/department-of-veterans-affairs/vets-api/pull/4220. It just hadn't been registered in the frontend.